type-definitions/dom-event.d.ts

```diff
@@ -0,0 +1,46 @@
+import {Stream} from "most";
+
+export function domEvent(event: string, node: EventTarget, capture: boolean): Stream<Event>
```

Comment:
Is there a way in TS to represent that `capture` is optional, and defaults to `false` when not provided?  Perhaps `capture:boolean = false` works, or perhaps just `capture = false` (hopefully TS could infer that it's a boolean!)?